src/schemaConfig.js

```diff
@@ -121,14 +122,18 @@ export function buildEndpoint(endpoint, params, options) {
   const usedParams = [];
   const paramEndpoint = endpoint.replace(/{(\w+)}/g, (match, key) => {
     usedParams.push(key);
-    return params[key];
+    return _.get(params, [key], '');
   });
+
   const unusedParamKeys = _.difference(Object.keys(params), usedParams);
-  const queryParams = unusedParamKeys.map(key => `${key}=${params[key]}`);
-  if (_.isEmpty(queryParams)) {
-    return paramEndpoint;
-  }
-  return `${paramEndpoint}?${queryParams.join('&')}`;
+  const queryParams = _.zipObject(
+    unusedParamKeys,
+    _.map(unusedParamKeys, key => params[key])
+  );
```

Comment:
It is somewhat hard to understand what is happening here, a comment would be nice, or you could probably use pick (https://lodash.com/docs/4.17.4#pick):
```
const queryParams = _.pick(params, unusedParams);
```